docs/admin/troubleshooting_updates.rst

```diff
@@ -104,8 +104,50 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    for assistance.
 
 4. Reboot the system. ``dom0`` updates are often
-   security-sensitive, and may requrie a reboot to take
+   security-sensitive, and may require a reboot to take
    effect.
+   
+GPG check failures
+^^^^^^^^^^^^^^^^^^
+
+If at any point you see a message that the GPG check failed,
```

Comment:
This is a bit vague, it should include the exact error text encountered with the RPM bug and the scenario in which users are likely. Otherwise it's possible that folks will follow said instructions for any key error, which we don't want, as it may be a legitimate failure (due to breakage or malicious behaviour) which we should find out about.

Reply:
Thanks @zenmonkeykstop! I just made it a bit more explicit, please let me know if you think there's anything more we can add it improve it.